### ISSUE TITLE
ransition to an rk1808 appears

### ISSUE THREAD

**Issue by bielolopez** [rockchip-linux/rknn-toolkit#429]
Because, when I run a test, this transition to an rk1808 appears, since it is a tinker edge r with rk3399pro.

"
--> Loading model
done
--> Building model
W The target_platform is not set in config, using default target platform rk1808.
W:tensorflow:From /root/.local/lib/python3.7/site-packages/tensorflow_core/python/framework/func_graph.py:915: py_func (from tensorflow.python.ops.script_ops) is deprecated and will be removed in a future version.
Instructions for updating:
tf.py_func is deprecated in TF V2. Instead, there are two
    options available in V2.
    - tf.py_function takes a python function which manipulates tf eager
    tensors instead of numpy arrays. It's easy to convert a tf eager tensor to
    an ndarray (just call tensor.numpy()) but having access to eager tensors
    means `tf.py_function`s can use accelerators such as GPUs as well as
    being differentiable using a gradient tape.
    - tf.numpy_function maintains the semantics of the deprecated tf.py_func
    (it is not differentiable, and manipulates numpy arrays). It drops the
    stateful argument making all functions stateful.
    
W:tensorflow:From /root/.local/lib/python3.7/site-packages/tensorflow_core/python/ops/resource_variable_ops.py:1630: calling BaseResourceVariable.__init__ (from tensorflow.python.ops.resource_variable_ops) with constraint is deprecated and will be removed in a future version.
Instructions for updating:
If using Keras pass *_constraint arguments to layers.
done
--> Export RKNN model
"

**Comment by eRaul** [comment]
These are TensorFlow warnings and can be ignored directly.
